typings/windows-process-tree.d.ts

```diff
@@ -24,11 +26,10 @@ declare module 'windows-process-tree' {
     pid: number;
     name: string;
     memory?: number;
+    commandLine?: string;
     children: IProcessTreeNode[];
   }
 
-
-
   /**
    * Returns a tree of processes with the rootPid process as the root.
    * @param {number} rootPid - The pid of the process that will be the root of the tree.
```

Comment:
You should omit the types here as they are redundant and could succumb to bit rot